xds/src/test/java/io/grpc/xds/XdsLoadBalancerProviderTest.java

```diff
@@ -73,9 +73,15 @@ public String getPolicyName() {
     public LoadBalancer newLoadBalancer(Helper helper) {
       return fakeBalancer1;
     }
+
+    @Override
+    public ConfigOrError parseLoadBalancingPolicyConfig(
+        Map<String, ?> rawLoadBalancingPolicyConfig) {
+      return ConfigOrError.fromConfig(lbConfig1);
+    }
   };
 
-  private final LoadBalancerProvider roundRobin = new LoadBalancerProvider() {
+  private final LoadBalancerProvider roundRobinProvider = new LoadBalancerProvider() {
```

Comment:
You can directly use real default registry with round_robin included.

Reply:
I need register `lbProvider1` as well. The default registry is global, it's not a good practice to change its state in test.

Reply:
That's ok, [`AutoConfiguredLoadBalancerFactoryTest`](https://github.com/grpc/grpc-java/blob/5ba663bcf6bcfd60c27d1e91a902eab19dffdc65/core/src/test/java/io/grpc/internal/AutoConfiguredLoadBalancerFactoryTest.java#L104-L105) is doing that thing.

Reply:
That is bad.

Reply:
No, repeating implementation is bad...